tasks/set-parse-version.yml

```diff
@@ -1,17 +1,17 @@
 ---
-- name: Convert datadog_agent_major_version to string
+- name: Convert agent_datadog_agent_major_version to string
```

Comment:
We actually convert `datadog_agent_major_version`, not `agent_datadog_agent_major_version`, so let's fix that in the task name.

Reply:
Again, I don't think this is actually resolved.